client/lib/application/src/approval-form/approval-form.js

```diff
@@ -39,7 +39,15 @@ class ApprovalForm extends BaseView {
         this
         .actions
         .saveApproval(applicationId, versionId, approval)
-        .then(() => this.actions.fetchApprovals(applicationId, versionId));
+        .then(() => this.actions.fetchApprovals(applicationId, versionId))
+        .catch(err => {
+            this
+            .props
+            .notificationActions
+            .addNotification(
+                `Could not approve version ${version_id} of ${this.data.application.name}. ${err.message}`,
```

Comment:
Agree.

----- Ursprüngliche Nachricht -----
Von: "Emanuel Lauria" <notifications@github.com>
Gesendet: ‎19.‎05.‎2015 17:58
An: "zalando-stups/yourturn" <yourturn@noreply.github.com>
Cc: "Nikolaus Piccolotto" <prayerslayer@gmail.com>
Betreff: Re: [yourturn] Update approvals (#103)

In client/lib/application/src/approval-form/approval-form.js:
> @@ -39,7 +39,15 @@ class ApprovalForm extends BaseView {
>          this
>          .actions
>          .saveApproval(applicationId, versionId, approval)
> -        .then(() => this.actions.fetchApprovals(applicationId, versionId));
> +        .then(() => this.actions.fetchApprovals(applicationId, versionId))
> +        .catch(err => {
> +            this
> +            .props
> +            .notificationActions
> +            .addNotification(
> +                `Could not approve version ${version_id} of ${this.data.application.name}. ${err.message}`,
should be versionId
—
Reply to this email directly or view it on GitHub.